packages/triple-web-binding-nextjs-pages/src/user-agent.ts

```diff
@@ -10,5 +11,6 @@ export function getUserAgent(ctx: NextPageContext): UserAgentValue {
 
   return {
     ...parser.getResult(),
+    isMobile: isMobile(userAgent),
```

Comment:
```suggestion
    isMobile: isMobile(parser),
```
타입 보니까 이거로 넘겨줘야 하는거 아닌가요?

Reply:
앗 맞아요. 수정했습니다!